apps/hash-graph/lib/graph/src/store/postgres/knowledge/entity.rs

```diff
@@ -124,7 +101,7 @@ impl<C: AsClient> PostgresStore<C> {
                     },
                     intersected_temporal_axes.clone(),
                 )
-                    .await?;
```

Comment:
How did `rustfmt` not pick these up before 🤔

Reply:
Rustfmt currently fails as soon as it sees `let Some(foo) = bar else {};`. Removing that line, applying `rustfmt`, adding it back solved this. AFAIK the rustfmt team is aware of this.